### ISSUE TITLE
Page Titled - 2.4.2 Pulling in Example from the WCAG 2.1 Page not the WCAG 2.2 Page

### ISSUE THREAD

**Issue by Grunet** [Grunet/digestible-wcag#9]
https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html#examples is the WCAG 2.1 it's pulling from, https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html#examples is what it should be pulling from.

Very unsure as to how this is happening. 

Issue is most probably in the scraping code

Needs more investigation.

**Comment by Grunet** [comment]
Unclear what happened here but re-running the scraper as-is fixed it.

Hunch is that the WCAG 2.2 understanding pages had leftover content from 2.1 and that was fixed after I updated the emails to 2.2
